src/components/custom/_signup-affiliate-details.js

```diff
@@ -18,7 +18,8 @@ import { localize, LocalizedLink } from 'components/localization'
 import device from 'themes/device.js'
 
 const StyledContentFlex = styled(Flex)`
-    width: 486px;
+    width: fit-content;
+    max-width: 486px;
```

Comment:
This should be min-width/height. How will the container expand dynamically if the max is set the the width 😐 Same for the other instances.